### ISSUE TITLE
Add more kittens for sandwich

### ISSUE THREAD

**Issue by lauriii** [DrupalTwig/sandwich#4]
So we should find a way how to add more kittens for this Drupal module. Every Drupal module needs kittens.

**Comment by star-szr** [comment]
@joelpittet and I have been talking about coming up with a different example module as well. Suggestions welcome, especially if they have something to do with kittens.

**Comment by joelpittet** [comment]
Yeah no more sandwich!  How about a theme function that uses a I can haz cheesburger phrase generator and overlays it kitten photos randomly:)

**Comment by lauriii** [comment]
+1 theme functions. I think theme functions is the thing in Drupal 8!
